test_ffi/tests/issue18131.js

```diff
@@ -0,0 +1,25 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+const targetDir = Deno.execPath().replace(/[^\/\\]+$/, "");
+const [libPrefix, libSuffix] = {
+  darwin: ["lib", "dylib"],
+  linux: ["lib", "so"],
+  windows: ["", "dll"],
+}[Deno.build.os];
+const libPath = `${targetDir}/${libPrefix}test_ffi.${libSuffix}`;
+
+const lib = Deno.dlopen(libPath, {
+  sleep_nonblocking: {
+    name: "sleep_blocking",
+    parameters: ["u64"],
+    result: "void",
+    nonblocking: true,
+  },
+});
+
+const ONE_HOUR = 1000 * 60 * 60;
```

Comment:
Don’t you think this is too time-consuming for cases when the test might fail?

Reply:
Agreed, 1 minute should be plenty.